### PR TITLE
Re-enable write permissions when installing with read-only-installdir

### LINKED ISSUE #3033: using "eb --skip --force" to install additional extensions fails if original installation was done with "--read-only-installdir"
For example, when installing additional extensions for `R-bundle-Bioconductor:

```
Error: ERROR: no permission to install to directory /software/R-bundle-Bioconductor/3.9-foss-2019a-R-3.6.0
```

### PR DESCRIPTION
**Pull request by Flamefire**
This allows e.g. adding additional packages (e.g. Python).
Fixes #3033

### CODE REVIEW

File: test/framework/toy_build.py [hunk 1]
```diff
@@ -130,11 +130,11 @@ def check_toy(self, installpath, outtxt, version='0.0', versionprefix='', versio
         # make sure installation log file and easyconfig file are copied to install dir
         software_path = os.path.join(installpath, 'software', 'toy', full_version)
         install_log_path_pattern = os.path.join(software_path, 'easybuild', 'easybuild-toy-%s*.log' % version)
-        self.assertTrue(len(glob.glob(install_log_path_pattern)) == 1, "Found 1 file at %s" % install_log_path_pattern)
+        self.assertTrue(len(glob.glob(install_log_path_pattern)) >= 1, "Found 1 file at %s" % install_log_path_pattern)
```
Comment (smoors) [inline test/framework/toy_build.py:133]:
```suggestion
        self.assertTrue(len(glob.glob(install_log_path_pattern)) >= 1, "Found  at least 1 file at %s" % install_log_path_pattern)
```

File: test/framework/toy_build.py [hunk 2]
```diff
@@ -130,11 +130,11 @@ def check_toy(self, installpath, outtxt, version='0.0', versionprefix='', versio
...
 
         # make sure test report is available
         test_report_path_pattern = os.path.join(software_path, 'easybuild', 'easybuild-toy-%s*test_report.md' % version)
-        self.assertTrue(len(glob.glob(test_report_path_pattern)) == 1, "Found 1 file at %s" % test_report_path_pattern)
+        self.assertTrue(len(glob.glob(test_report_path_pattern)) >= 1, "Found 1 file at %s" % test_report_path_pattern)
```
Comment (smoors) [inline test/framework/toy_build.py:137]:
```suggestion
        self.assertTrue(len(glob.glob(test_report_path_pattern)) >= 1, "Found  at least 1 file at %s" % test_report_path_pattern)
```

**Review by smoors — COMMENTED** [review]
looks good to me, just a small suggestion to change the assert strings

File: test/framework/toy_build.py [hunk 3]
```diff
@@ -130,11 +130,11 @@ def check_toy(self, installpath, outtxt, version='0.0', versionprefix='', versio
         # make sure installation log file and easyconfig file are copied to install dir
         software_path = os.path.join(installpath, 'software', 'toy', full_version)
         install_log_path_pattern = os.path.join(software_path, 'easybuild', 'easybuild-toy-%s*.log' % version)
-        self.assertTrue(len(glob.glob(install_log_path_pattern)) == 1, "Found 1 file at %s" % install_log_path_pattern)
+        self.assertTrue(len(glob.glob(install_log_path_pattern)) >= 1, "Found  at least 1 file at %s" % install_log_path_pattern)
 
         # make sure test report is available
         test_report_path_pattern = os.path.join(software_path, 'easybuild', 'easybuild-toy-%s*test_report.md' % version)
-        self.assertTrue(len(glob.glob(test_report_path_pattern)) == 1, "Found 1 file at %s" % test_report_path_pattern)
+        self.assertTrue(len(glob.glob(test_report_path_pattern)) >= 1, "Found  at least 1 file at %s" % test_report_path_pattern)
```
[the remark this answers is not in the document]
Comment (Flamefire) [reply test/framework/toy_build.py:137]:
```suggestion
        self.assertTrue(len(glob.glob(test_report_path_pattern)) >= 1,
                              "Should find at least 1 file at %s" % test_report_path_pattern)
```

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @Flamefire!

**Pull request merged**